vpr/src/route/connection_router.cpp

```diff
@@ -911,12 +914,10 @@ void ConnectionRouter<Heap>::add_route_tree_node_to_heap(
 
     if (!rcv_path_manager.is_enabled()) {
         // tot_cost = backward_path_cost + cost_params.astar_fac * expected_cost;
+        float expected_cost = router_lookahead_.get_expected_cost(inode, target_node, cost_params, R_upstream);
         float tot_cost = backward_path_cost
                          + cost_params.astar_fac
-                               * router_lookahead_.get_expected_cost(inode,
-                                                                     target_node,
-                                                                     cost_params,
-                                                                     R_upstream);
+                               * std::max(0.f, expected_cost - cost_params.astar_offset);
```

Comment:
Now that get_expected_cost() is not called here, is it necessary to break this down into 3 lines?